store_app/business/connect_external_API.py

```diff
@@ -0,0 +1,40 @@
+from urllib.request import urlopen
+import json
+
+api_key_path = 'keys/API_Key.txt'
+
```

Comment:
It's a shame in the end you refrained from using classes. The whole beauty of OOP is that you could group all these functions into a class (as methods). This takes the structure of your project to the next level as well as long term it allows for better control of dependencies. 

Also if you find a good name for the class, you can reuse that name for the file itself (not a fan of the current name - this is effectively an API gateway, and more specifically OmdbApiGateway)

Reply:
I will try to implement it using classes